_docs/3.0.0/usage/spark-jobs-quick-start.md

```diff
@@ -0,0 +1,186 @@
+<spark home>/spark-submit \
+--num-executors 2 \
+--executor-memory 2G \
+--master yarn \
+--deploy-mode cluster \
+--driver-cores 2 \
+--driver-memory 2G \
+--class za.co.absa.enceladus.standardization.StandardizationJob \
+spark-jobs_<build_version>.jar \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <report_version> \
+--raw-format <data_format> \
+--row-tag <tag>
+```
+
+where:
+
+- `dataset_name` is the name given the dataset per [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start) guide Guide
+- `dataset_version` is a version of the dataset to use, which should have the correct schema and all the desired conformance rules 
+- `report_date` represents the date on which the data landed in the HDFS (in raw) and also the suffix part of the data output path. So if in Menas raw was specified as `/path/on/hdfs/raw` and the input parameter `report_date` as `2020-12-24` then the path where standardization will look for input files will be `/path/on/hdfs/raw/2020/12/24`. For the final part we are missing the report versions.
+- `report_version` is the final part of the path on HDSF. With `report_date` we finished with `/path/on/hdfs/raw/2020/12/24/v<report_version>`. This is the location where standardization will look for raw data.
+- `raw-format` and its specifics. Raw format tells the standardization which format the data is in on the HDFS and what are its specifics. CSV might have a header, XML has a row-tag, etc. Here in the example, we use the `row-tag`. For more options for different types and run parameters see our [run documentation](run) or just run `--help`
+
+
+### Running Conformance
+
+```shell
+<spark home>/spark-submit \
+--num-executors 2 \
+--executor-memory 2G \
+--master yarn \
+--deploy-mode cluster \
+--driver-cores 2 \
+--driver-memory 2G \
+--class za.co.absa.enceladus.conformance.DynamicConformanceJob \
+spark-jobs_<build_version>.jar \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <data_run_version>
+```
+
+### Running Standardization and Conformance together
+
+```shell
+<spark home>/spark-submit \
+--num-executors 2 \
+--executor-memory 2G \
+--master yarn \
+--deploy-mode cluster \
+--driver-cores 2 \
+--driver-memory 2G \
+--class za.co.absa.enceladus.standardization_conformance.StandardizationAndConformanceJob \
+spark-jobs_<build_version>.jar \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <report_version> \
+--raw-format <data_format> \
+--row-tag <tag>
+```
+
+Here, nothing new is added for the quick run. Of course, there might be special options which are all documented in the [run documentation](https://absaoss.github.io/enceladus/docs/3.0.0/usage/run)
```

Comment:
```suggestion
Here, nothing new is added for the quick run. Of course, there might be special options which are all documented in the [run documentation](run)
```

Perhaps the link to run documentation could be relative, the same way as on line 48 - that way it works when deployed elsewhere (even when debugging locally), too.